go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/Azure/eraser
 
-go 1.16
```

Comment:
hmm CI is failing for everything, can you try updating to 1.17

Reply:
👍